Webscraper e interface de teste

GCyrillo opened the pull request:
Arquivo python na pasta webscraper. HTML e CSS são só para teste

Commit:
Webscraper e interface de teste

ET33 commented on webscraper/webscraper.py:
```diff
@@ -0,0 +1,59 @@
+
+def getMoreInfo(url, data):
+	soup = load(url)
+	links = soup.select(".caixa-noticia-categoria")
```
`.caixa-noticia-categoria` poderia ser um parâmetro para generalizar :D

ET33 reviewed (approved):
:shipit:

Commit:
Comentários e mudança no main.js

Tinha uma parte do código que eu não lembro o porque estava lá. Deixei comentado pois vai que seja útil no futuro
